gslib/commands/signurl.py

```diff
@@ -162,6 +162,13 @@
                Note that both options have a maximum allowed duration of
                12 hours for a valid link.
 
+  -b <project> Allows you to specify a user project that will be billed for
```

Comment:
I think generally the options are listed alphabetically for gsutil commands, though I see that's not the case currently for this command

Reply:
I can scoot these around so that they're alphabetical.